src/DynamoRevit/DynamoRevitApp.cs

```diff
@@ -381,8 +381,41 @@ public static bool DynamoButtonEnabled
             set;
         }
 
-        private bool TryResolveDynamoCore()
+        /// <summary>
+        /// Whether the DynamoCore and DynamoRevit are in Revit's Addin, 
```

Comment:
to you mean to say `in Revit's Addin folder`?

Reply:
Yes, it should be in Revit's internal addin folder.